cli/src/main.rs

```diff
@@ -440,12 +440,14 @@ async fn main() -> Result<()> {
             cancel,
         } => {
             let drain = !cancel;
-            nats.request(&DrainDrone {
-                cluster: ClusterName::new(&cluster),
-                drone: DroneId::new(drone),
-                drain,
-            })
-            .await?;
+            for drone in drone {
+                nats.request(&DrainDrone {
+                    cluster: ClusterName::new(&cluster),
+                    drone: DroneId::new(drone),
+                    drain,
+                })
+                .await?;
```

Comment:
Instead of aborting on the first error, let's just report the error and move on to the next drone. That way, if your connection is dropped mid-way through draining a bunch of drones, you can just run the same command again.